### PR TITLE
Adding the stipulation about subresource loading to "out of scope"

### PR DESCRIPTION
**Pull request by zachernuk**
per discussion in the CG meeting today!

***
<a href="https://pr-preview.s3.amazonaws.com/immersive-web/model-element/pull/90.html" title="Last updated on Jul 15, 2024, 6:55 PM UTC (1646b86)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/immersive-web/model-element/90/1b25a02...1646b86.html" title="Last updated on Jul 15, 2024, 6:55 PM UTC (1646b86)">Diff</a>

### CODE REVIEW

**Comment by zachernuk** [comment]
@LaszloGombos does this reflect the discussion we had in the last CG meeting? Happy to revise but if these PRs look okay I've got more to add! (I just don't want to overload the process with all of them at once)

File: index.html [hunk 1]
```diff
@@ -101,6 +101,14 @@ <h3>
         aware of such actions, stateful interactions should be considered out
         of scope.
       </p>
+      <h3>
+        Subresource loading
+      </h3>
+      <p>
+        While many popular model formats support the use of external asset
+        requests for better re-use and consistency, this initial specification
```
Comment (cabanier) [inline index.html:109]: I would phrase it stronger:
this specification will only allow assets that are self-contained

Comment (zachernuk) [reply index.html:109]: Perfect. updated!

Comment (LaszloGombos) [reply index.html:109]: looks great

**Review by LaszloGombos — APPROVED** [review]
lgtm

**Pull request merged**